src/index.js

```diff
@@ -1,33 +1,89 @@
 import './style.css';
 
 const toDoList = document.querySelector('#toDoList');
+let selectedIndex;
 
-const list = [
-  {
-    description: 'have fun',
-    completed: false,
-    id: 1,
-  },
-  {
-    description: 'study a lot',
-    completed: false,
-    id: 2,
-  },
-  {
-    description: 'learn more',
-    completed: false,
-    id: 3,
-  },
-];
+if (!localStorage.getItem('todo')) {
+  localStorage.setItem('todo', JSON.stringify([
+    {
+      description: 'have fun',
+      completed: false,
+      id: 1,
+    },
+    {
+      description: 'study a lot',
+      completed: false,
+      id: 2,
+    },
+    {
+      description: 'learn more',
+      completed: false,
+      id: 3,
+    },
+  ]));
+}
+const handleGetData = () => {
+  const list = localStorage.getItem('todo');
+  if (list) {
+    return JSON.parse(list);
+  }
+  return [];
+};
+
+const draghandler = ({ i }) => {
+  selectedIndex = i;
+};
+const dragoverhandler = (ev) => {
+  ev.preventDefault();
+};
+const drophandler = ({ i }) => {
+  const list = handleGetData();
+  const tmp = list[selectedIndex];
+  list[selectedIndex] = list[i];
+  list[i] = tmp;
+  localStorage.setItem('todo', JSON.stringify(list));
+  window.location.reload();
+};
+const dragstarthandler = () => {
 
```

Comment:
- [ ] As the requirements mentioned the project is supposed to have 2 new files imported as modules
  - One should contain methods related to drag / drop and sorting functionality.
  - The other should contain methods related to the status updates (`completed`: `true` / `false`). 
  Please work on that!